server/tracedb/datastoreresource/resource_types.go

```diff
@@ -202,21 +223,26 @@ func (ds DataStore) GetID() id.ID {
 }
 
 func (ds DataStore) IsOTLPBasedProvider() bool {
+	if ds.Type == DatastoreTypeAzureAppInsights {
```

Comment:
In the future, I'd prefer having each integrator decide if they are otlp or not based on the values and such. this patch is temporary